ateam_kenobi/src/path_planning/path_planner.cpp

```diff
@@ -51,7 +51,7 @@ PathPlanner::Path PathPlanner::getPath(
   }
 
   if (options.avoid_ball) {
-    augmented_obstacles.push_back(ateam_geometry::makeCircle(world.ball.pos, 0.04267 / 2));
+    augmented_obstacles.push_back(ateam_geometry::makeDisk(world.ball.pos, 0.04267 / 2));
```

Comment:
While we are modifying these files we might as well change this constant over to `kBallDiameter`